javageci-core/src/main/java/javax0/geci/equals/Equals.java

```diff
@@ -243,7 +244,7 @@ public void process(Source source, Class<?> klass, CompoundParams global, Field[
     }
 
     private void generateHashCodeBody(Segment segment, CompoundParams global, Field[] fields) {
-        segment.write("int result = 0;");
+        segment.write("int result = super.hashCode();");
```

Comment:
Same as for equals.

If the parent is `Object` two objects will only have the same hashCode if they are one object.